### PR TITLE
added shell=True in process.run for pci and multipath file.

### PR DESCRIPTION
**Pull request by smruti77**
for fixing this error, ERROR: [Errno 8] Exec format error
added shell=True in script.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

File: avocado/utils/multipath.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def device_exists(path):
     :return: True if path exists, False if does not exist.
     """
     cmd = "multipath -l %s" % path
-    if process.system(cmd, ignore_status=True, sudo=True) != 0:
+    if process.system(cmd, ignore_status=True, sudo=True, shell=True) != 0:
```
Comment (PraveenPenguin) [inline avocado/utils/multipath.py]: ` if process.system(cmd, ignore_status=True, sudo=True, shell=True):` should be sufficent

File: avocado/utils/multipath.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def get_multipath_wwids():
     """
     cmd = "egrep -v '^($|#)' /etc/multipath/wwids"
     wwids = process.run(cmd, ignore_status=True,
-                        sudo=True).stdout_text
```
Comment (clebergnu) [inline avocado/utils/multipath.py]: @smruti77  I don't follow the reason for this change.  For instance:

```
$ python3
>>> cmd = "egrep -v '^($|#)' /etc/multipath/wwids"


>>> process.run(cmd)
<avocado.utils.process.CmdResult object at 0x7faa506ca950>
>>> process.run(cmd).stdout_text
'/0QEMU_QEMU_HARDDISK_3600508b400105e21000/\n'



>>> process.run(cmd, shell=True)
<avocado.utils.process.CmdResult object at 0x7faa506d5490>
>>> process.run(cmd, shell=True).stdout_text
'/0QEMU_QEMU_HARDDISK_3600508b400105e21000/\n'
```

I don't see why a shell is needed here and in all the commands I tried.  Do you have a reproducer?  Or maybe the reviewers @beraldoleal @PraveenPenguin can help me with that too.

Comment (beraldoleal) [reply avocado/utils/multipath.py]: Yes, @clebergnu I see your point and I agree with it. Maybe just a try... except will be enough here.

@smruti77, do you have the exact error that you are getting?

**Comment by PraveenPenguin** [comment]
@clebergnu ah yes you are right my Bad as it can be system (enviornment ) issue as well as when I tried that i also faced exec error but now not able to reproduce :(  @beraldoleal i agree with you we can have try expect block

**Comment by smruti77** [comment]
@beraldoleal @clebergnu i got this error in pci.py, ERROR: [Errno 8] Exec format error (lspci -ks 8001:01:00.0): 'lspci' . so i changed in pci and i chnaged for mutipath.

**Comment by clebergnu** [comment]
@smruti77 did you get this in `pci.py` in a specific environment? The following works fine on my environment:

```
>>> from avocado.utils import process
>>> cmd = 'lspci -ks 8001:01:00.0'
>>> process.run(cmd)
<avocado.utils.process.CmdResult object at 0x7f725b8be1d0>
>>> process.run(cmd).stdout_text
''
```

About the try/except block, if we want to ignore the nonzero status code, we can just use `ignore_status` parameter.  Now, about the `[Errno 8] Exec format error` this is how I can reproduce it:

```
$ touch /tmp/empty
$ chmod +x /tmp/empty

$ python3
>>> from avocado.utils import process
>>> process.run('/tmp/empty')

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/cleber/src/avocado/avocado/avocado/utils/process.py", line 1036, in run
    cmd_result = sp.run(timeout=timeout)
  File "/home/cleber/src/avocado/avocado/avocado/utils/process.py", line 909, in run
    self._init_subprocess()
  File "/home/cleber/src/avocado/avocado/avocado/utils/process.py", line 623, in _init_subprocess
    raise details
  File "/home/cleber/src/avocado/avocado/avocado/utils/process.py", line 620, in _init_subprocess
    env=self.env)
  File "/usr/lib64/python3.7/subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.7/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
OSError: [Errno 8] Exec format error (/tmp/empty): '/tmp/empty'
```

Which doesn't seem like something we want to catch and ignore.

**Pull request merged**